api/events.proto

```diff
@@ -57,4 +100,58 @@ service Events {
       body: "*"
     };
   };
+  // Like the Stream rpc, but then scoped to Application events.
+  rpc StreamApplication(StreamApplicationEventsRequest) returns (stream Event) {
```

Comment:
@bafonins do we actually **need** these versions of the RPC? It would be a lot easier to maintain the codebase if we could make everything work with the original RPC (so `POST /api/v3/events`) instead of having (almost) duplicate RPCs

Reply:
Yes. See https://github.com/TheThingsNetwork/lorawan-stack/issues/28#issuecomment-481146020 regarding the `POST` method in `oboe.js`. Unfortunately, `oboe` is the only library I was able to find that supports streaming responses in both nodejs/browsers, and making `POST` request is broken there atm.

Reply:
It doesn't feel right to adapt our API to work around a bug in a library we're using. @johanstokking what do you think we should do here?

Reply:
indeed. i can fork the library and fix the issue

Reply:
Is this still relevant?